drivers/java/src/main/java/com/rethinkdb/net/Connection.java

```diff
@@ -345,7 +345,7 @@ public void runNoReply(ReqlAst term, OptArgs globalOpts) {
 
     void stop(Cursor cursor) {
```

Comment:
Someone changed this and i just change back. This method is only used by Cursor.java currently.